src/librustc_privacy/lib.rs

```diff
@@ -229,12 +229,15 @@ impl<'a, 'tcx> Visitor<'tcx> for EmbargoVisitor<'a, 'tcx> {
             hir::ItemUse(..) => {}
             // The interface is empty
             hir::ItemGlobalAsm(..) => {}
-            hir::ItemExistential(..) => {
+            hir::ItemExistential(hir::ExistTy { impl_trait_fn: Some(_), .. }) => {
                 if item_level.is_some() {
                     // Reach the (potentially private) type and the API being exposed
-                    self.reach(item.id).ty().predicates();
+                    self.reach(item.id).predicates();
```

Comment:
The comment wasn't updated. Is the intention here that the "type" part doesn't need to be exported, unlike the "predicates", and the reachability pass should handle this instead?